src/gpgmm/d3d12/ResidencyManagerD3D12.cpp

```diff
@@ -568,10 +568,34 @@ namespace gpgmm::d3d12 {
     // Given a list of heaps that are pending usage, this function will estimate memory needed,
     // evict resources until enough space is available, then make resident any heaps scheduled for
     // usage.
+    // /Deprecated use ResidencyList instead of ResidencySet
```

Comment:
Might need to to escape this comment like `/** \brief  ... */` and have `\deprecated` (might be case-sensitive).

Reply:
Done. 

I probably didn't need to add this comment in the .cpp file since it is in the .h...but it's not hurting anything I guess.